proto/crosschain/cross_chain_tx.proto

```diff
@@ -4,6 +4,7 @@ package zetachain.zetacore.crosschain;
 import "common/common.proto";
 import "gogoproto/gogo.proto";
 
+//TODO : fix the descriptor numbers for the fields
```

Comment:
What does it mean? The numbers?
Can we provide more details and track it with an issue?

Reply:
40c8aac0e5a12174b19876c6f36cf76288edf2c0